setup.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from setuptools import find_packages, setup
+
+setup(
+    name='cmsplugin-articles-ai',
+    version='0.0.1',
+    author='Anders Innovations',
+    author_email='info@anders.fi',
+    packages=find_packages(),
```

Comment:
❗ This will include all packages in the distribution, incl. tests and so on, and that's not what we want.